packages/side-nav/src/utils.ts

```diff
@@ -24,3 +24,15 @@ export const localPalette = {
 	linkHoverBg: `var(${localPaletteVars.linkHoverBg})`,
 	linkActiveBg: `var(${localPaletteVars.linkActiveBg})`,
 };
+
+// Content that should be read out
+// to screenreaders only
+export const visuallyHiddenStyles = {
```

Comment:
Can we make this a component instead? `<VisuallyHidden>{children}</VisuallyHidden>`

Reply:
Or a prop on Box / Text